src/components/SignerPanel/SignerPanel.js

```diff
@@ -31,6 +31,11 @@ const INITIAL_STATE = {
 
 const RECIEPT_ERROR_STATUS = '0x0'
 
+const getPretransactionDescription = intent =>
+  `Allow ${intent.name.toLowerCase()} to ${intent.description
```

Comment:
We generally leave the name of the app capitalized; I think `intent.name` should be fine since that's the name that app developers give us